### PR TITLE
Add gabikas name into project

### PR DESCRIPTION
**Pull request by Embi**
because Gabika is cool

### CODE REVIEW

File: includes/solution.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+gabika
```
Comment (Embi) [inline includes/solution.php]: Meno ma zacinat s velkym pismenom.

Comment (lukassiarny) [inline includes/solution.php]: sdasdasdsadsa

File: includes/solution.php [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+Gabika
```
Comment (lukassiarny) [inline includes/solution.php]: dfsdfsdfdsfsdf

File: includes/solution.php (hunk 1 above)
Comment (Embi) [reply includes/solution.php]: Opravil som to

File: includes/solution.php (hunk 2 above)
Comment (Embi) [reply includes/solution.php]: kjashdksajdh

Comment (lukassiarny) [reply includes/solution.php]: Ok, moze byt

**Pull request merged**